aporia/assets/dashboards/aporia_alerts.json

```diff
@@ -0,0 +1,141 @@
+{
+    "title": "Aporia Alerts",
+    "description": null,

```

Comment:
Please include a description in the dashboard that includes the value of the dashboard